doc/tutorial/single_cell_allen.rst

```diff
@@ -0,0 +1,170 @@
+.. _tutorialsinglecellallen:
+
+A single cell model from the Allen Brain Atlas
+==============================================
+
+In this tutorial we'll see how we can take a model from the `Allen Brain Atlas <https://brain-map.org/>`_
+and run it with Arbor.
+
+.. Note::
+
+   **Concepts covered in this example:**
+
+   1. Take a model from the Allen Brain Atlas.
+   2. Load a morphology from an ``swc`` file.
+   3. Load a parameter fit file and apply it to a :class:`arbor.decor`.
+   4. Building a :class:`arbor.cable_cell` representative of the cell in the model.
+   5. Building a :class:`arbor.recipe` reflective of the cell stimulation in the model.
+   6. Running a simulation and visualising the results.
+
+Obtaining the model
+-------------------
+
+We need a model for which morphological data is available. We'll take
+`a cell from the mouse's visual cortex <http://celltypes.brain-map.org/experiment/electrophysiology/488683425>`_.
```

Comment:
No HTTPS :/ Also, to isolate us against mishaps we should cache the downloaded file, eg in arbor-contrib.

Reply:
Never trust ;)

Reply:
I get your point, but it's the Allen DB. If those URLs become invalid, millions of papers will cry out in terror and be suddenly silent right along with us.

Reply:
True, we could make it N-1 and we use this particular example quite often (L2L, here, Posters, ...). But fine if you think it's too much.

Reply:
We supply the bits we use for the example. As with all references: let's fix them as they break.